recipe/meta.yaml

```diff
@@ -33,13 +33,13 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     # llvm is needed for the headers
-    - llvm-openmp              # [osx]
-
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
   host:
     - python
     - pip
     - setuptools
-    - llvmlite 0.40.*
+    - llvmlite 0.41.*
     - numpy
     - tbb-devel 2021.6.0
 
```

Comment:
Since we dropped the blank line above, maybe we can make this consistent and do the same here

```suggestion
```